### ISSUE TITLE
configuration file, especially for fullscreen & window resolution

### ISSUE THREAD

**Issue by vsariola** [vsariola/sointu#184]
#180 suggested recalling the previous window setting when reopening the tracker. But that approach does not work when there is nothing to recall, e.g., when first opening the VST instrument in a DAW. For those situations, we might want have a ~ default settings for the tracker user configurable.

The straightforward approach would be to have a .yml file in the Sointu config directory, which contains the configuration. For example, something like:
```
fullscreen: true
```

Or

```
window-width: 1920
window-height: 1080
```

But things to consider:

1) Would all the keyboard shortcuts be in this file too? Or are they in a separate file, as they are now?

2) Would we have different config file for the VST and the tracker?

3) How does this play with the idea #180? Do we need even #180 if we have this config file?

4) Are there any other settings that would be good to be user configurable?

**Comment by qm210** [comment]
Howdy, as said in https://github.com/vsariola/sointu/pull/185
sorry, didn't see this issue earlier, so the structure is for now (easily changeble, I have no hard opinions here)

```
window:
  width: 1600
  height: 900
  maximized: true
```

I kept the key bindings separate (because of their sheer volume), but I put these under the "window" substructure as I would agree that other preferences can live there just as they want to :)

**Comment by vsariola** [comment]
Implemented in #185 / 7ff3c94, closing for now. We can add more Preferences as need.
